gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/sampling/functions/EdgeTargetVertexJoin.java

```diff
@@ -25,29 +24,38 @@
 /**
  * Joins to get the edge target
  */
-public class EdgeTargetVertexJoin implements JoinFunction<Tuple3<Edge, Vertex, GradoopId>,
-        Tuple2<Vertex, GradoopId>, Tuple3<Edge, Vertex, Vertex>> {
+public class EdgeTargetVertexJoin implements
```

Comment:
Maybe add a description of the join: `(edge,edge.targetId,bool),(vertex) -> (edge,bool,(bool)vertex[propertyKey])`
Also add an annotation that the edge is forwarded